server/pom.xml

```diff
@@ -97,6 +97,7 @@
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-auth</artifactId>
+            <scope>${hadoop.scope}</scope>
```

Comment:
They are transitive dependencies of hadoop-client.
I changed hadoop-client's scope below.